src/app/api/common/delegates/getDelegates.ts

```diff
@@ -135,7 +135,62 @@ async function getDelegates({
   seed?: number;
 }) {
   const pageSize = 20;
-  const { namespace } = Tenant.current();
+  const { namespace, ui } = Tenant.current();
+
+  const allowList = ui.delegates?.allowed || [];
+  const hasAllowList = allowList.length > 0;
+
+  // Applies allow-list filtering to the delegate list
+  const paginatedAllowlistQuery = async (skip: number, take: number) => {
+    switch (sort) {
+      case "most_delegators":
+        return prisma.$queryRawUnsafe<DelegatesGetPayload[]>(
+          `
+            SELECT *
+            FROM ${namespace + ".delegates"}
+            WHERE num_of_delegators IS NOT NULl AND delegate = ANY($1)
```

Comment:
does this `NULl` spacing issue matter at all

Reply:
good catch, strange...